src/librustdoc/html/static/css/themes/ayu.css

```diff
@@ -630,5 +630,5 @@ input:checked + .slider {
 	background: #616161;
 }
 .toggle-line:hover .toggle-line-inner {
-	background: ##898989;
+	background: #898989;
```

Comment:
cc @willcrichton – this will change the display of scrape-examples

Reply:
Ah yeah this is fixed in my PR.